backend/src/main/java/gov/cdc/usds/simplereport/service/PatientBulkUploadService.java

```diff
@@ -35,186 +33,128 @@
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
-/** Created by nickrobison on 11/21/20 */
+/**
+ * Service to upload a roster of patient data given a CSV input. Formerly restricted to superusers
+ * but now (almost) available to end users.
+ *
+ * <p>Updated by emmastephenson on 10/24/2022
+ */
 @Service
 @Transactional
 @RequiredArgsConstructor
 @Slf4j
 public class PatientBulkUploadService {
-  private static final int MAX_LINE_LENGTH = 1024 * 6;
-  public static final String ZIP_CODE_REGEX = "^[0-9]{5}(?:-[0-9]{4})?$";
 
-  private final PersonService personService;
-  private final AddressValidationService addressValidationService;
-  private final OrganizationService organizationService;
-  private boolean hasHeaderRow = false;
+  private final PersonService _personService;
+  private final AddressValidationService _addressValidationService;
+  private final OrganizationService _organizationService;
+  private final PatientBulkUploadFileValidator _patientBulkUploadFileValidator;
 
-  private MappingIterator<Map<String, String>> getIteratorForCsv(InputStream csvStream)
+  // This authorization will change once we open the feature to end users
+  @AuthorizationConfiguration.RequireGlobalAdminUser
+  public PatientBulkUploadResponse processPersonCSV(InputStream csvStream, UUID facilityId)
       throws IllegalArgumentException {
+
+    PatientBulkUploadResponse result = new PatientBulkUploadResponse();
+    result.setStatus(UploadStatus.FAILURE);
```

Comment:
Moved! I was originally thinking it's best to set the failure status at the top in case any unexpected exceptions are thrown, but since we handle those at the controller level it's a bit of a moot point.